### PR TITLE
add splashradius property to PopupMenuButton

### PR DESCRIPTION
**Pull request by maheshj01**
Addresses https://github.com/flutter/flutter/issues/96206

*List which issues are fixed by this PR. You must list at least one issue.*

*If you had to change anything in the [flutter/tests] repo, include a link to the migration guide as per the [breaking change policy].*

## Pre-launch Checklist

- [X] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [X] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [X] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [X] I signed the [CLA].
- [X] I listed at least one issue that this PR fixes in the description above.
- [X] I updated/added relevant documentation (doc comments with `///`).
- [X] I added new tests to check the change I am making, or this PR is [test-exempt].
- [X] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: packages/flutter/lib/src/material/popup_menu.dart
Comment (Hixie) [inline packages/flutter/lib/src/material/popup_menu.dart:1100]:
```suggestion
  /// If this property is null, the default splash radius is [Material.defaultSplashRadius].
```

File: packages/flutter/test/material/popup_menu_test.dart
Comment (Hixie) [inline packages/flutter/test/material/popup_menu_test.dart:156]: should also test that the default is correct

**Review by Hixie — APPROVED** [review]
LGTM modulo the minor comments i left

This will need a second review before landing.

File: packages/flutter/test/material/popup_menu_test.dart
Comment (maheshmnj) [reply packages/flutter/test/material/popup_menu_test.dart:159]: done

**Comment by maheshmnj** [comment]
Closing as this got landed https://github.com/flutter/flutter/issues/90898

**Pull request closed without merging**
